src/libs/antares/benchmarking/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+set(SRC_BENCHMARKING
```

Comment:
Note: we can probably also use this PR to remove the unused source file `file_writer.cpp` in the benchmarking directory.